Makefile

```diff
@@ -42,7 +42,7 @@ cold-reboot: contracts ## Run the cold reboot test
 
 .PHONY: test contracts cold-reboot assert-counter-contract-small
 
-MAX_COUNTER_CONTRACT_SIZE = 8192
+MAX_COUNTER_CONTRACT_SIZE = 32768
```

Comment:
This is a **huge** difference. Should we make two different `wrap_call`s available? Say `wrap_call` and `wrap_call_unchecked`?

Reply:
sounds good, I will do that